### PR TITLE
Added a thank you for Elvis

### PR DESCRIPTION
**Pull request by sbramlett**

### CODE REVIEW

File: scripts/thanks.coffee [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  "no, no. thank you!",
+  "sure thing"
+]
+
+module.exports = (robot) ->
+  robot.respond /thank(s| you)/i, (msg) ->
+    msg.send msg.random response
+  thanks = new RegExp "thank(s| you) #{robot.name}", "i"
+  robot.hear thanks, (msg) ->
+msg.send msg.random response
```
Comment (dpritchett) [inline scripts/thanks.coffee]: probably need to tab this over to be two columns to the right of line 39

Comment (dpritchett) [reply scripts/thanks.coffee]: actually all of this needs to be hierarchically indented.  everything below line 36 two columns to the right of 36 and then line 40 needs to be 2 columns even further right than 39

**Comment by dpritchett** [comment]
image

**Comment by lilithlavender** [comment]
I'm ok with with the way you're doing this, as it still requires the robot to be mentioned by name. @dpritchett am I correct in reading your screenshot that one example is `thanks elvis` and the other is `@elvis thanks`, both of which are explicitly responding to the bot's name?

**Comment by dpritchett** [comment]
That's the way I read it as well.  I spammed `thanks` a bunch and it didn't
reply.

image

**Comment by lilithlavender** [comment]
Good enough for me

**Pull request merged**